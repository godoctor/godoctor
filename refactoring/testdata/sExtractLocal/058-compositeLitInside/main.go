package main

import "fmt"

func main() {
	comp := Class{5}
	pr := &comp
	tree := School{*pr, Books{"/ab", "false", "/ab", "nil"}} // <<<<< extractLocal,8,44,8,49,newVar,pass
	fmt.Println("there's a name in there")
	for _, index := range tree.Books {
		fmt.Println(index)
	}
}

type School struct {
	Class
	Books
}
type Class struct {
	classNum int
}

type Books []string
