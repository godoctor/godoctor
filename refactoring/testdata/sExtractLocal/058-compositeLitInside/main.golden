package main

import "fmt"

func main() {
	comp := Class{5}
	pr := &comp
	newVar := "/ab"
	tree := School{*pr, Books{"/ab", "false", newVar, "nil"}} // <<<<< extractLocal,8,44,8,49,newVar,pass
	fmt.Println("there's a name in there")
	for _, index := range tree.Books {
		fmt.Println(index)
	}
}

type School struct {
	Class
	Books
}
type Class struct {
	classNum int
}

type Books []string
